<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QMainWindow" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MindMaze - Menu</string>
  </property>
  <property name="windowIcon">
   <iconset resource="textures/textures.qrc">
    <normaloff>:/maze.ico</normaloff>:/maze.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="BoxSize">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>281</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Difficulté</string>
    </property>
    <widget class="QSlider" name="SliderWidth">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>160</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>25</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>0</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
     <property name="tickInterval">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="LabelWidth">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Largeur</string>
     </property>
    </widget>
    <widget class="QLabel" name="LabelHeight">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Hauteur</string>
     </property>
    </widget>
    <widget class="QSlider" name="SliderHeight">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>160</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>25</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>0</number>
     </property>
     <property name="value">
      <number>6</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="TextWidth">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TextHeight">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QSlider" name="SlidernbSphere">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="pageStep">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="LabelnbSphere">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>47</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>nbSphere</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TextnbSphere">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>281</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Réglages</string>
    </property>
    <widget class="QSlider" name="SliderFOV">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>160</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>30</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="pageStep">
      <number>0</number>
     </property>
     <property name="value">
      <number>85</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="LabelFOV">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>FOV</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TextFOV">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>85</string>
     </property>
    </widget>
    <widget class="QSlider" name="SliderVolume">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>160</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="pageStep">
      <number>0</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="LabelVolume">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>volume</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TextVolume">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLabel" name="LabelResolution">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Resolution :</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CheckFullscreen">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Plein Ecran</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="TextWinWidth">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1600</string>
     </property>
    </widget>
    <widget class="QLabel" name="LabelX">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>90</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="TextWinHeight">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>900</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>171</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Controles</string>
    </property>
    <widget class="QCheckBox" name="CheckSnapping">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>72</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="(plus simple)">permet de ne se déplacer que sur le centre des cases
et de tourner de 90degrés par mouvement (plus simple)</string>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="text">
      <string>snapping</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="CheckKeyboard">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>72</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="(plus simple)">permet de ne se déplacer que sur le centre des cases
et de tourner de 90degrés par mouvement (plus simple)</string>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="text">
      <string>clavier</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="CheckCamera">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>72</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="(plus simple)">permet de ne se déplacer que sur le centre des cases
et de tourner de 90degrés par mouvement (plus simple)</string>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="text">
      <string>camera</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="CheckMouse">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>72</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="(plus simple)">permet de ne se déplacer que sur le centre des cases
et de tourner de 90degrés par mouvement (plus simple)</string>
     </property>
     <property name="toolTipDuration">
      <number>-1</number>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="text">
      <string>souris</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>260</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>JOUER</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>564</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufichier">
    <property name="title">
     <string>fichier</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menufichier"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="textures/textures.qrc"/>
 </resources>
 <connections/>
</ui>
